ui_components.gemspec

```diff
@@ -30,20 +37,15 @@ Gem::Specification.new do |s|
   s.add_dependency 'bootstrap_form'
   s.add_dependency 'rails_bootstrap_navbar'
   s.add_dependency 'bootstrap-datepicker-rails'
-  s.add_dependency 'rails-assets-bootstrap-daterangepicker', '< 2'
-  s.add_dependency 'rails-assets-bootstrap-chosen'
-  s.add_dependency 'rails-assets-marked'
-  s.add_dependency 'rails-assets-ajax-chosen'
+  s.add_dependency 'bower-rails'
 
   s.add_dependency 'jquery-turbolinks'
   s.add_dependency 'jquery-rails'
   s.add_dependency 'underscore-rails'
```

Comment:
Most of them are tied to some sort of Rails magic. I'll leave that for another day.